src/main/java/com/wolox/training/exceptions/RestExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.wolox.training.exceptions;
+
+import javax.validation.ConstraintViolationException;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class RestExceptionHandler extends ResponseEntityExceptionHandler {
+
+    public static final String BOOK_ID_MISMATCH = "Book ID mismatch";
```

Comment:
Its a good practice have all constants messages in a separate class

Reply:
where do you recommend to create this class?